Integrations Health dashboard bug fix- show errors right after fresh install (#11608)
demisto/content: Packs/IntegrationsAndIncidentsHealthCheck/ReleaseNotes/1_2_0.md

> 

Reviewer: please follow https://confluence.paloaltonetworks.com/display/DemistoContent/Release+Notes+And+Descriptions

#### Scripts
##### IntegrationsCheck-Widget-NumberFailingInstances

> - Fix an issue with IntegrationsErrorsInfo widget that shows error after fresh install.

Reviewer 2: ```suggestion
- Fix an issue where the *IntegrationsErrorsInfo* widget would show an error after a fresh install.
```

##### IntegrationsCheck-Widget-IntegrationsErrorsInfo

> - Fix an issue with  NumberFailingInstances widget that shows 1 failed instance after fresh install while there is no fail instances. 

Reviewer 2: ```suggestion
- Fix an issue where the *NumberFailingInstances* widget would display 1 failed instance after a fresh install when there were no failed instances. 
```
